### DISCUSSION TITLE
Full Title Is Not Showing Up on Hovering After Upgrade

### DISCUSSION THREAD

**Discussion by cryptogun** [lawand-dot-io/taskbar#191] [Feature requests]
**Describe the bug**
I have multiple apps and Chromes open so full titles can't be seen.
In a previous version of taskbar v1, when I hover onto an app, the full title would popup.
In the latest version of taskbar v1, nothing shows up on hovering.
P.S. I haven't give recording permission for thumbnails' creation.

**To Reproduce**
1. Create multiple window of Chrome.
2. Disable thumbnail functionality.
3. Hover onto any window within taskbar.
4. No full-title popup will be shown.

**Expected behavior**
Full title should popup on hovering.

**Desktop (please complete the following information):**
 - Taskbar version [1.0]
 - OS: [macOS 15.1]
 - Chip [M1]

**Comment by lawand-dot-io** [comment]
Hello. This feature wasn't properly supported in previous versions. It used to be that hovering will sometimes show a tool-tip and sometimes not, and the text wasn't always correct, so I removed the feature and I plan to implement it correctly in newer versions. Hope that was clear. Cheers
